toolkit/scripts/toolchain/container/toolchain_build_in_chroot.sh

```diff
@@ -781,22 +781,23 @@ popd
 rm -rf openssl-1.1.1g
 touch /logs/status_openssl_complete
 
-echo Python-3.7.4
-tar xf Python-3.7.4.tar.xz
-pushd Python-3.7.4
+echo Python-3.9.9
+tar xf Python-3.9.9.tar.xz
+pushd Python-3.9.9
 ./configure --prefix=/usr       \
+            --with-platlibdir=lib \
             --enable-shared     \
             --with-system-expat \
             --with-system-ffi   \
             --with-ensurepip=yes
```

Comment:
Not sure it makes a difference, but I think you switched this to `no` in the spec.

Reply:
That's intentional, the `ensurepip` process works fine here but fails in RPM build environment (there should be some links for context in the spec).